bflib/src/db/actions.rs

```diff
@@ -1676,6 +1754,35 @@ impl Db {
         }
     }
 
+    fn cruise_missile_mission<'lua>(
+        &mut self,
+        side: Side,
+        ucid: Option<Ucid>,
+        spawn_pos: Vector2,
+        args: WithPosAndGroup<()>,
+    ) -> Result<Vec<MissionPoint<'lua>>> {
+        let group = group!(self, args.group)?;
+        let init_task = Task::ComboTask(vec![]);
+        let main_task = if group.tags.contains(UnitTag::Link16) {
+            vec![Task::WrappedCommand(Command::SetUnlimitedFuel(true))]
+        } else {
+            vec![Task::WrappedCommand(Command::SetUnlimitedFuel(true))]
+        };
```

Comment:
Why do both branches of this if do the same thing. This is actual aids. If you want the tu160 to have unlimited fuel then just set unlimited fuel.